apps/web/src/app/(signing)/sign/[token]/provider.tsx

```diff
@@ -44,6 +44,12 @@ export const SigningProvider = ({
   const [email, setEmail] = useState(initialEmail || '');
   const [signature, setSignature] = useState(initialSignature || null);
```

Comment:
This wouldn't initialize `signature` to `initialSignature` all the time, so I had to use the `useEffect` below.

Reply:
This is because the parent component is no longer key'ing it which used to be the case. So the useState will retain the value until it's unmounted even if the prop changes